model/src/cmd/ModelCommandAPI.cc

```diff
@@ -0,0 +1,437 @@
+    }
+    {
+
+      std::map<sdf::JointType, std::string> jointTypes = {
+        {sdf::JointType::REVOLUTE, "revolute"},
+        {sdf::JointType::BALL, "ball"},
+        {sdf::JointType::CONTINUOUS, "continuous"},
+        {sdf::JointType::FIXED, "fixed"},
+        {sdf::JointType::GEARBOX, "gearbox"},
+        {sdf::JointType::PRISMATIC,  "prismatic"},
+        {sdf::JointType::REVOLUTE2, "revolute2"},
+        {sdf::JointType::SCREW, "screw"},
+        {sdf::JointType::UNIVERSAL, "universal"}
+      };
+
+      auto jointComp =
+          ecm.Component<ignition::gazebo::components::JointType>(_entity);
+      auto childLinkComp =
+          ecm.Component<ignition::gazebo::components::ChildLinkName>(_entity);
+      auto parentLinkComp =
+          ecm.Component<ignition::gazebo::components::ParentLinkName>(_entity);
+
+      if (childLinkComp && parentLinkComp)
+      {
+        std::cout << "    - Joint type:  " << jointTypes[jointComp->Data()]
+        << "\n"   << "    - Parent Link: [" << childLinkComp->Data() << "]\n"
+                  << "    - Child Link:  [" << parentLinkComp->Data() << "]\n";
+      }
+    }
+  }
+}
+
+//////////////////////////////////////////////////
+void printBoundingBox(uint64_t entity,
+                ignition::gazebo::EntityComponentManager &ecm)
+{
+  auto bounding_boxes = ecm.EntitiesByComponents(
+  ignition::gazebo::components::ParentEntity(entity),
+  ignition::gazebo::components::AxisAlignedBox());
+
+  for(auto &_entity : bounding_boxes)
+  {
+    {
+      auto parentComp =
+          ecm.Component<ignition::gazebo::components::ParentEntity>(_entity);
+
+      auto nameComp =
+          ecm.Component<ignition::gazebo::components::Name>(_entity);
+
+      std::string parentInfo;
+      if (parentComp)
+      {
+        auto parentNameComp =
+            ecm.Component<ignition::gazebo::components::Name>(
+            parentComp->Data());
+
+        if (parentNameComp)
+        {
+          parentInfo += parentNameComp->Data() + " ";
+        }
+        parentInfo += "[" + std::to_string(parentComp->Data()) + "]";
+      }
+
+      std::cout << "  - Bounding Box [" << _entity << "]" << std::endl
+                << "    - Name: " << nameComp->Data() << std::endl
+                << "    - Parent: " << parentInfo << std::endl;
+    }
+    {
+      auto boxComp =
+          ecm.Component<ignition::gazebo::components::AxisAlignedBox>(_entity);
+
+      if (boxComp)
+      {
+
+        std::cout << "     Size:   [" << boxComp->Data().Size() << "]\n"
+                  << "     Center: [" << boxComp->Data().Center() << "]\n"
+                  << "     Min:    [" << boxComp->Data().Min() << "]\n"
+                  << "     Max:    [" << boxComp->Data().Max() << "]\n";
+      }
+    }
+  }
+}
+
+//////////////////////////////////////////////////
+extern "C" IGNITION_GAZEBO_VISIBLE void cmdModelInfo(
+  const char *_model, int _pose, int _link, const char *_link_name,
```

Comment:
Check style for the name of the parameters

Reply:
Addressed